packages/translation-extension/src/index.ts

```diff
@@ -99,7 +99,14 @@ const langMenu: JupyterFrontEndPlugin<void> = {
       .then(setting => {
         // Read the settings
         loadSetting(setting);
-        document.documentElement.lang = (currentLocale ?? '').replace('_', '-');
+
+        // Ensure currentLocale is not 'default' which is not a valid language code
+        if (currentLocale !== 'default') {
+          document.documentElement.lang = (currentLocale ?? '').replace(
+            '_',
+            '-'
+          );
+        }
```

Comment:
```suggestion
        }
        else {
          document.documentElement.lang = 'en-US';
        }
```

Thank you for working on this! Based on the [MDN recommendation](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang):

> Note: The default value of lang is unknown, therefore it is recommended to always specify this attribute with the appropriate value.

I think we should set it to `en-US` if it is `default` rather than leaving it hanging.

Reply:
That's a good point, thank you